backend/typescript/services/implementations/camperService.ts

```diff
@@ -128,8 +136,39 @@ class CamperService implements ICamperService {
         throw new Error("All sessions must belong to the same camp");
       }
 
+      // Ensure all campers meet the age requirements
+      if (
+        !campers.every(
+          (camper) =>
+            camper.age >= camp.ageLower && camper.age <= camp.ageUpper,
+        )
+      ) {
+        throw new Error("Not all campers meet the age requirements");
+      }
+
+      // Add the total charges for the campers:
+      const lineItems = createStripeLineItems(
```

Comment:
do we need these line items later at some point or can we access them through the checkout session?

Reply:
Not sure we'll need them elsewhere?